src/docs/asciidoc/hazelcast_clients.adoc

```diff
@@ -304,13 +304,13 @@ The equivalent programmatic approach is shown below.
 ----
 ClientConfig clientConfig = new ClientConfig();
 clientConfig.setInstanceName("ExampleClientName");
-clientConfig.setAttribute("role", "user");
-clientConfig.setAttribute("region", "bar");
+clientConfig.addLabel("user");
+clientConfig.addLabel("bar");
 
 HazelcastClient.newHazelcastClient(clientConfig);
 ----
 
-See the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients/client-attributes[code sample] for the client attributes to see them in action.
+See the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients/client-labels[code sample] for the client attributes to see them in action.
```

Comment:
... for the client labels to see them in action.

Reply:
@ihsandemir , @sancar , the "attribute" usage in the reference manual will not create a confusion right?

Reply:
done